src/Nethermind/Nethermind.Network/Rlpx/ZeroFrameMerger.cs

```diff
@@ -16,14 +16,20 @@ public class ZeroFrameMerger : ByteToMessageDecoder
     {
         private ILogger _logger;
 
-        private ZeroPacket _zeroPacket;
+        private ZeroPacket? _zeroPacket;
         private FrameHeaderReader _headerReader = new();
 
         public ZeroFrameMerger(ILogManager logManager)
         {
             _logger = logManager?.GetClassLogger<ZeroFrameMerger>() ?? throw new ArgumentNullException(nameof(logManager));
         }
 
+        public override void HandlerRemoved(IChannelHandlerContext context)
+        {
+            base.HandlerRemoved(context);
+            _zeroPacket?.Release();
```

Comment:
Are you sure it isn't released anywhere else?

Reply:
Yes